caluma/caluma_form/api.py

```diff
@@ -4,6 +4,19 @@
 from caluma.caluma_user.models import BaseUser
 
 
+def _get_new_answer(LogicClass, data, user, answer):
```

Comment:
This could probably be implemented as a `@classmethod` on `SaveAnswerLogic` instead, so you don't have to do your own polymorphism

Reply:
Not really. It's used in `save_answer()` and `save_default_answer()`.

Reply:
Yes, but IMO it'd be cleaner to call like `SaveAnswerLogic.get_new_answer()` and `SaveDefaultAnswerLogic.get_new_answer()` than the current calls. Since it's a class method, you can get the correct class implicitly

Reply:
You're right. Done!